app/src/main/java/io/branch/branchster/SplashActivity.java

```diff
@@ -34,6 +39,21 @@ protected void onCreate(Bundle savedInstanceState) {
         FacebookSdk.sdkInitialize(getApplicationContext());
         AppEventsLogger.activateApp(getApplicationContext());
 
+        AppLinkData.fetchDeferredAppLinkData(getApplicationContext(), new AppLinkData.CompletionHandler() {
+            @Override
+            public void onDeferredAppLinkDataFetched(AppLinkData appLinkData) {
+                if (appLinkData != null && appLinkData.getArgumentBundle() != null) {
+                    JSONObject obj = new JSONObject();
+                    try {
+                        obj.put("applink", appLinkData.getArgumentBundle().toString());
+                    } catch (JSONException e) {
+                        e.printStackTrace();
+                    }
+                    Branch.getInstance().userCompletedAction("fetched_app_link", obj);
```

Comment:
this shouldn't be a problem, as init will be in flight or very likely, completed. 👍